src/main/java/com/algorand/algosdk/v2/client/model/BoxesResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.algorand.algosdk.v2.client.common.PathResponse;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Box names of an application
+ */
+public class BoxesResponse extends PathResponse {
```

Comment:
This response does not contain a `next-token` field, which is needed for indexer pagination

Reply:
@jasonpaulos It seems codegen source code is out of date.  I re-generated via https://github.com/algorand/java-algorand-sdk/pull/401.  Can you have a look?